drape/overlay_handle.hpp

```diff
@@ -165,7 +168,7 @@ class OverlayHandle
   bool IsReady() const { return m_isReady; }
 
   void SetDisplayFlag(bool display) { m_displayFlag = display; }
-  bool GetDisplayFlag() const { return m_displayFlag; }
+  bool GetDisplayFlag() const { return true; } // @todo Remove all displayFlag logic as its not used anymore.
```

Comment:
Ok, if we modify here, so worth removing m_displayFlag var, no?

Reply:
Yes, I will remove it in subsequent PRs together with associated logic.

Reply:
Then I have doubts, what do we fix here? I see that functions:
```
handle->SetDisplayFlag(false);
handle->SetDisplayFlag(true);
```
are called, and the logic looks valid.